docfiles/tracking.html

```diff
@@ -9,10 +9,25 @@
                 sdk.addTelemetryInitializer(function (envelope) {
                     // App Insights automatically sends a page view event on setup, but we send our own later with additional properties.
                     // This stops the automatic event from firing, so we don't end up with duplicate page view events.
-                    if(envelope.baseType == "PageviewData" && !envelope.baseData.properties) {
+                    if (envelope.baseType == "PageviewData" && !envelope.baseData.properties) {
                         return false;
                     }
 
+                    if (envelope.baseType == "PageviewPerformanceData") {
+                        var pageName = envelope.baseData.name;
```

Comment:
Why is `var` used throughout instead of `const`? I don't see any reassignment.

Reply:
I was just matching the rest of the file since this one's going to be ported to several editors; this one's directly embedded in the page (not ts or anything) so we've historically been pretty cautious of using newfangled javascript features, though I believe we have stopped having to support editors that this would have caused problems in so should be fine to move over to let / const's across the board now (though I'd leave that for a separate pr most likely, just for ease of porting)